conf/evolutions/default/88.sql

```diff
@@ -0,0 +1,12 @@
+CREATE OR REPLACE FUNCTION add_proposals_from_theme_to_wg(theme_id bigint, wg_id bigint) RETURNS void AS $$
+    DECLARE
+
+    BEGIN
+      insert into resource_space_working_groups
+        select resource_space_resource_space_id, wg_id from contribution where resource_space_resource_space_id in (
+        select resource_space_resource_space_id
+        from resource_space_theme
+        where theme_theme_id = theme_id
```

Comment:
@yolile I think we might get duplicated contributions because the same contribution might be associated to more than one theme. Can you add a way of avoiding this?